pkg/tarmak/cluster/cluster.go

```diff
@@ -859,12 +859,8 @@ func (c *Cluster) PublicAPIHostname() string {
 	)
 }
 
-// retrieve Amazons EBS encryption status
-func (c *Cluster) AmazonEBSEncrypted() bool {
-	if c.Config().Amazon != nil && c.Config().Amazon.EBSEncrypted != nil && *c.Config().Amazon.EBSEncrypted {
-		return true
-	}
-	return false
+func (c *Cluster) Encrypted() bool {
+	return *c.conf.Amazon.EBSEncrypted
```

Comment:
What guarantees do we have this is not nil?

Reply:
This shouldn't be because of defaults but I can add a check